SignalMessaging/utils/ThreadUtil.h

```diff
@@ -44,27 +44,32 @@ NS_ASSUME_NONNULL_BEGIN
 #pragma mark -
 
 @class TSOutgoingMessage;
+@class TSQuotedMessage;
 
 @interface ThreadUtil : NSObject
 
 + (TSOutgoingMessage *)sendMessageWithText:(NSString *)text
                                   inThread:(TSThread *)thread
+                             quotedMessage:(nullable TSQuotedMessage *)quotedMessage
                              messageSender:(OWSMessageSender *)messageSender
                                    success:(void (^)(void))successHandler
                                    failure:(void (^)(NSError *error))failureHandler;
 
 + (TSOutgoingMessage *)sendMessageWithText:(NSString *)text
                                   inThread:(TSThread *)thread
+                             quotedMessage:(nullable TSQuotedMessage *)quotedMessage
                              messageSender:(OWSMessageSender *)messageSender;
 
 + (TSOutgoingMessage *)sendMessageWithAttachment:(SignalAttachment *)attachment
                                         inThread:(TSThread *)thread
+                                   quotedMessage:quotedMessage
```

Comment:
Aren't you missing a type here?

Reply:
Sure am. I didn't even realize that could compile.